plugins/woocommerce/woocommerce.php

```diff
@@ -16,6 +16,12 @@
 
 defined( 'ABSPATH' ) || exit;
 
+add_filter( 'plugin_configuration_data_woocommerce', static function() {
```

Comment:
The filter to be added in WordPress 6.5.4 was renamed to include a prefix. See [WP61269 (comment)](https://core.trac.wordpress.org/ticket/61269#comment:50)

```suggestion
add_filter( 'wp_plugin_configuration_data_woocommerce', static function() {
```